doc/doxygen/src/mainpage.md

```diff
@@ -23,6 +23,10 @@ Across all supported hardware (32-bit, 16-bit, and 8-bit platforms). RIOT
 provides a consistent API and enables ANSI C and C++ application programming,
 with  multithreading, IPC, system timers, mutexes etc.
 
+[This article](https://riot-os.org/docs/riot-ieeeiotjournal-2018.pdf), published
+in the IEEE Internet of Things Journal, volume 5 issue 6, provides a good
```

Comment:
how about:
A good high-level overview can be found in the article [RIOT: An Open Source Operating System for Low-End Embedded Devices in the IoT](https://riot-os.org/docs/riot-ieeeiotjournal-2018.pdf) (IEEE Internet of Things Journal, December 2018).

Reply:
Done